app/models/application.rb

```diff
@@ -33,27 +34,18 @@ class Application < ActiveRecord::Base # rubocop:disable ClassLength
       order('payments.expires_at ASC')
   }
 
+  # Fixme remove this delegation methods when all tests are clean
+  APPLICANT_GETTERS = %i[title first_name last_name date_of_birth ni_number married married?]
+  APPLICANT_SETTERS = %i[title= first_name= last_name= date_of_birth= ni_number= married=]
+  delegate(*APPLICANT_GETTERS, to: :applicant)
+  delegate(*APPLICANT_SETTERS, to: :applicant)
+
   MAX_AGE = 120
   MIN_AGE = 16
 
   after_save :run_auto_checks
-  before_validation :format_ni_number
   before_validation :nullify_blank_emergency_reason
 
-  # Step 1 - Personal detail validation
-  with_options if: proc { active_or_status_is? 'personal_information' } do
-    validates :last_name, presence: true
-    validates :married, inclusion: { in: [true, false] }
-    validates :last_name, length: { minimum: 2 }, allow_blank: true
-    validates :date_of_birth, date: true
-    validate :dob_age_valid?
```

Comment:
If these are deleted here, where are they being validated now?  https://github.com/ministryofjustice/fr-staffapp/pull/388/files#diff-1e3f6eb69dfed2e3702dad512eb74667R6 only seems to validate the NI Number below.